health/health.d/megacli.conf

```diff
@@ -66,6 +66,5 @@ component: RAID
     units: cycles
     every: 10s
      warn: $this >= 100
-     crit: $this >= 500
```

Comment:
Why removing?

It is a recommendation from Intel ([according to the user](https://github.com/netdata/netdata/pull/3930#issuecomment-402994606)) to **replace** the battery after recharging cycles. So critical makes total sense to me. I think we need it.

Reply:
Is it something I need to do at the middle of the night? :) we could go with making warning on this. Also it varies on what battery do you have , are you charging it to 100% and keeping it there constantly or just 70-80%?

Reply:
It is something that reports a different problem (warn vs crit) with a different solution/action. So it should be a separate alarm or crit. For me, crit looks ok here.

Reply:
Please elaborate. Thing is that I always think in the way that critical means - I need to go there and do it regardless of day or time. What is the sense of critical alert here? Ok we have batteries that are old - but still it means that this critical is going to be there at best for a week - you need to order the batteries. Do you really think that UPS should have red light on it because of it?
I shall not insist on that - If I did not convince you at this point it is clear that you as a client would need it so I should add this back.

Reply:
The sense of critical here is 500+ is a point when you should have already replaced the battery. I am not sure that would be a reason for you to wake up but it would be for me.